### DISCUSSION TITLE
Security Onion 2.4 - SOC Missing under so-status

### DISCUSSION THREAD

**Discussion by smallboy69** [Security-Onion-Solutions/securityonion#12057] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

20

### RAM

256gb

### Storage for /

9TB

### Storage for /nsm

9TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

hi Community

so i installed 2.4.30 on dec 8 on Oracle 9.3 via downloaded ISO image and it was all working fine services

just recently i notice so-soc saying missing

have attached the logs  and only symptom i can see is initialization error 

appreciate your insight and experience as experts on Security onion

Thanks
**edited to remove file

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment]
Please use a text file or a gist for the logs. Word docs don't get opened.

**Reply by smallboy69** [reply]
please find attached error message in portion of log
[Uploading sensoroni-server-20231219-Event.txt…]()

**Reply by dougburks** [reply]
@smallboy69 That hyperlink just points to this discussion:
```
https://github.com/Security-Onion-Solutions/securityonion/discussions/12057
```

**Reply by smallboy69** [reply]
My apologise @dougburks 

new to using GitHub and did not verify correct link after i posted it.

my mistake

[sensoroni-server-20231219-Event.txt]
(https://github.com/Security-Onion-Solutions/securityonion/files/13746361/sensoroni-server-20231219-Event.txt)

**Reply by jertel** [reply, marked as answer]
Hi @smallboy69, it looks like a corrupted job file. Try removing this file from your manager node: `/nsm/soc/jobs/internalso/1002.json`.

**Reply by smallboy69** [reply]
thanks @jertel will action it and see if it resolves the issue

regards

**Reply by smallboy69** [reply]
Thanks @jertel it works 

![SOC](https://github.com/Security-Onion-Solutions/securityonion/assets/40338264/86c14254-c4d6-4647-9433-198f74f9413d)

Much appreciated
